diracx-db/pyproject.toml

```diff
@@ -27,15 +25,15 @@ testing = [
     "diracx-testing",
 ]
 
-[project.entry-points."diracx.db.sql"]
+[project.entry-points."diracx.dbs.sql"]
```

Comment:
I am confused: is it `db` or `dbs`?

Reply:
It's whatever we want as long as we are consistent.

We have:
- `project.entry-points."diracx.services"`
- `project.entry-points."diracx.access_policies"`

So I thought it would be more consistent to have every entry points using plural.